src/openzaak/conf/docker.py

```diff
@@ -5,6 +5,8 @@
 os.environ.setdefault("DB_USER", "postgres")
 os.environ.setdefault("DB_PASSWORD", "")
 
+os.environ.setdefault("LOG_STDOUT", "yes")
```

Comment:
In some places the values of `LOG_STDOUT` are inconsistent:
* here: default value = 'yes'
* in `docs/installation/logging.rst` and in `.env-example `: values = 0 or 1
* in django config: default value = false